common/src/main/java/whocraft/tardis_refined/common/tardis/TardisArchitectureHandler.java

```diff
@@ -70,7 +77,7 @@ public static void buildAirlockEntranceFromStructure(StructureTemplate template,
                     Optional<StructureTemplate> structureNBT = level.getLevel().getStructureManager().get(new ResourceLocation(TardisRefined.MODID, "corridors/airlock_entrance"));
                     structureNBT.ifPresent(structure -> {
                         BlockPos offsetPosition = new BlockPos(3, 2, 0);
-                        structure.placeInWorld(level.getLevel(), pos.subtract(offsetPosition), pos.subtract(offsetPosition), new StructurePlaceSettings(), level.getLevel().random, 3);
+                        structure.placeInWorld(level.getLevel(), pos.subtract(offsetPosition), pos.subtract(offsetPosition), new StructurePlaceSettings(), level.getLevel().random, 2);
```

Comment:
The "2" should be the flag field from the Block class, we are trying NOT to have magic numbers

Reply:
Updated